examples/event-migration/build.sbt

```diff
@@ -0,0 +1,26 @@
+name := "event-migration"
+version := "0.1"
+scalaVersion := "2.13.6"
+
+val circeVersion = "0.14.1"
+val borerVersion = "1.7.1"
```

Comment:
Hmmm 1.7.1 here vs 1.7.2 in the rest of the project?

Reply:
Theoretically doesn't need to be the same in examples and in the rest of the project... but still